app/views/guide_colour.html

```diff
@@ -234,12 +234,12 @@ <h4 class="heading-small">Mauve</h4>
         <li>$mauve</li>
       </ul>
 
-      <h4 class="heading-small">Fuschia</h4>
```

Comment:
Good catch. Might be worth splitting this change into a separate PR so it can be merged relatively quickly without some of the expected debate associated with the other changes.

Reply:
(Or we pick an easier to spell colour name – not that I can think of an alternative)